backend/routes/nested-path-example/db-row.js

```diff
@@ -0,0 +1,19 @@
+import pgp from "pg-promise";
+
+const db = pgp()(process.env.PG_CONNECTION_STRING);
```

Comment:
I left this in as an example, if we need to do any manual queries without the ORM. We probably won't use this, so we can take it out when we start adding the real routes